src/book.py

```diff
@@ -101,6 +131,7 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         garbage = '\ufeff|â€™|â€"|â€œ|â€˜|â€\x9d|â€œi|_|â€'
         cleaned = re.sub(garbage, '', cleaned)
         cleaned = cleaned.replace('-', ' ')
+        cleaned = re.sub(r'\n+', ' ', cleaned)
```

Comment:
Should we just make this all string characters, something akin to regex '/s'?

Reply:
We can. I will have to see how it behaves.

Reply:
I suggested something like this elsewhere in the code (Kyle, I assume you meant whitespace characters, not string characters)

Reply:
Ah yes, thanks for the catch. Regex is hard during group meetings.

Reply:
Regex is always hard

Reply:
Can someone else please look into removing all string characters? For now, I want to keep it this way, and maybe address '\s' in a new pull request.

Reply:
This is already included in line 154 (or is automatically done if lemmatizing), so I don't think more work is needed

Reply:
Ok. I am resolving this conversation then.